Policies/github-corporate-terms-of-service.md

```diff
@@ -38,42 +38,47 @@ Effective date: October 11, 2017
 2. The “Service” refers to the applications, software, products, and services provided by GitHub.
 3. The “Website” refers to GitHub’s website located at [github.com](https://github.com/), and all content, services, and products provided by GitHub at or through the Website. It also refers to GitHub-owned subdomains of github.com, such as [education.github.com](https://education.github.com/) and [pages.github.com](https://pages.github.com/). These Terms also govern GitHub’s conference websites, such as [githubuniverse.com](https://githubuniverse.com/), and product websites, such as [atom.io](https://atom.io/). Occasionally, websites owned by GitHub may provide different or additional terms of service. If those additional terms conflict with this Agreement, the more specific terms apply to the relevant page or service.
 4. "Customer", “You,” and “Your” refer to the company, or organization that has visited or is using the Website or Service; that accesses or uses any part of the account; or that directs the use of the account in the performance of its functions. Special terms may apply for business or government accounts (See [Section B(6): Additional Terms](#6-additional-terms)).
-5. "User" refers to the individual who has visited or is using the Website or Service on your behalf; that accesses or uses any part of the account on your behalf; or that directs the use of the account in the performance of their functions on your behalf. A User must be at least 13 years of age.
+5. "User" refers to the individual who has visited or is using the Website or Service on your behalf; that accesses or uses any part of the account on your behalf; or that directs the use of the account in the performance of their functions on your behalf. All Users must be at least 13 years of age. "Other Users" refers to the individuals, not including your Users, who visit or use the Website or Service. “Users” and “Other Users” are collectively referred to as “All Users”.
```

Comment:
Should the “All Users” definition go before its first use (“All Users must be at least 13 years of age”)?